spec/factories/tailoring.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :v2_tailoring, class: 'V2::Tailoring' do
+    profile do
+      V2::Profile
+        .joins(:os_minor_versions)
+        .find_by!(
+          ref_id: ref_id,
+          profile_os_minor_versions: {
+            os_minor_version: os_minor_version
+          }
+        )
+    end
+    value_overrides { profile.value_overrides }
+
+    transient do
+      ref_id { policy.profile.ref_id }
+      os_major_version { 7 }
+    end
+
+    trait :with_tailored_values do
+      value_overrides do
+        profile.value_overrides&.each do |k, v|
+          # overriding (tailoring) original values
+          profile.value_overrides[k] = v + SecureRandom.random_number(100)
+        end
+      end
+    end
+
+    trait :with_mixed_rules do
+      rules do
+        # sample of rules under profile
+        profile_rules = profile.rules.sample(profile.rules.count / 2)
+        # sample of rules under security guide, outside of the profile
+        sec_guide_rules_avail = security_guide.rules.reject { |rule| profile_rules.include?(rule) }
+        sec_guide_rules = sec_guide_rules_avail.sample(sec_guide_rules_avail.count / 2)
+
+        sec_guide_rules + profile_rules
+      end
+    end
+
+    trait :with_default_rules do
+      after(:create) do |tailoring, _|
+        tailoring.profile.rules.each do |rule|
+          FactoryBot.create(
+            :v2_tailoring_rule,
+            rule: rule,
+            tailoring: tailoring
+          )
+        end
+      end
```

Comment:
I think this should be the default behavior, and maybe there could be the `with_no_rules` trait as an edge case. Most users have default tailoring anyway so it makes sense to me. WDYT?

Reply:
Makes sense to me. Default rules should be the default here.